graphql/handler/transport/websocket.go

```diff
@@ -283,7 +287,9 @@ func (c *wsConnection) run() {
 		case pingMessageType:
 			c.write(&message{t: pongMessageType, payload: m.payload})
 		case pongMessageType:
-			c.conn.SetReadDeadline(time.Now().UTC().Add(2 * c.PingPongInterval))
+			c.receivedPong = true
```

Comment:
`receivedPong` probably doesn't need to be a member since it's only used in this loop and this thread, right?